src/relax/op/tensor/unary.cc

```diff
@@ -22,11 +22,60 @@
  * \brief unary operators.
  */
 
-#include "unary.h"
+#include "../op_common.h"
 
 namespace tvm {
 namespace relax {
 
+#define RELAX_REGISTER_UNARY_OP(OpName)                               \
+  TVM_REGISTER_GLOBAL("relax.op." OpName).set_body_typed([](Expr e) { \
+    static const Op& op = Op::Get("relax." OpName);                   \
+    return Call(op, {e}, Attrs(), {});                                \
+  });                                                                 \
+  RELAX_REGISTER_OP("relax." OpName)                                  \
```

Comment:
avoid the relax prefix, in all registration function, directly spell out full namr for clarity
```
RELAX_REGISTER_UNARY_OP("relax.negative")
```

Reply:
When registering the Make function to FFI, the key string is `"relax.op.op_name"`, which has one `".op"` in the middle more than the operator key `"relax.op_name"`.

One walk-around is to pass two full-name strings to the macro. And another might be having two macros.

Reply:
Get it. In this case not having relax prefix seems indeed was better then. Please add a
Quick comment on the macro side

Reply:
I see. Updated with some quick notes.